debugtools/DDR_VM/src/com/ibm/j9ddr/AuxFieldInfo29.dat

```diff
@@ -60,6 +60,8 @@ CompiledMethodWrapper.romMethodOffset = required
 GC_ArrayletObjectModelBase._arrayletRangeBase = required
 GC_ArrayletObjectModelBase._arrayletRangeTop = required
 GC_ArrayletObjectModelBase._largestDesirableArraySpineSize = required
+GC_ArrayletObjectModelBase._enableDoubleMapping = bool
+GC_ArrayletObjectModelBase._enableVirtualLargeObjectHeap = bool
```

Comment:
Please keep things ordered.

I don't think `bool` is supported (at least not on all platforms).

Reply:
Should I use `boolean` instead, or what would you recommend?

Reply:
Using `bool` is not a problem in this file, but I don't think compilers produce debug information for that type consistently across platforms. If you're only using in native code, there's no problem, but mentioning those fields is only necessary if you want to access them in DDR code in which case I won't be surprised if at least one platform has trouble. Have you done builds for all platforms yet?